src/ci/resources/dummy_application.conf

```diff
@@ -3,6 +3,7 @@ include "build_application.inc.conf"
 
 system.job-rate-control {
   jobs = 150
+  per = 1 second
```

Comment:
The dummy backend was making some unsafe assumptions about the denominator being 1 second which it was often not during my experiments